src/Services/IqRequestService.ts

```diff
@@ -34,8 +34,12 @@ export class IqRequestService {
   ) {}
 
   private async init(): Promise<void> {
-    this.internalId = await this.getApplicationInternalId();
-    this.isInitialized = true;
+    try {
+      this.internalId = await this.getApplicationInternalId();
+      this.isInitialized = true;
+    } catch (e) {
```

Comment:
haha was this to get rid of the unhandled promise errors?

Reply:
🤷‍♂

Reply:
Yeah, more or less!